tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -53,22 +201,130 @@ def check_extracted_metadata(self, metadata: dict):
-        )
+        assert metadata["NWBFile"]["session_start_time"] == datetime(2023, 9, 22, 12, 51, 34, 124000)
+
+
+@parameterized_class(
+    [
+        {
+            "channel_name": "Channel 1",
+            "photon_series_name": "TwoPhotonSeriesChannel1",
+            "imaging_plane_name": "ImagingPlaneChannel1",
+        },
+        {
+            "channel_name": "Channel 2",
+            "photon_series_name": "TwoPhotonSeriesChannel2",
+            "imaging_plane_name": "ImagingPlaneChannel2",
+        },
+    ],
+)
+class TestScanImageMultiFileImagingInterfaceSinglePlaneCase(ScanImageSinglePlaneImagingInterfaceMixin, TestCase):
+    data_interface_cls = ScanImageMultiFileImagingInterface
+    interface_kwargs = dict(
+        folder_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage"),
+        file_pattern="scanimage_20240320_multifile*.tif",
+    )
+    save_directory = OUTPUT_PATH
+
+    channel_name = "Channel 1"
+    photon_series_name = "TwoPhotonSeriesChannel1"
+    imaging_plane_name = "ImagingPlaneChannel1"
```

Comment:
```suggestion
    imaging_plane_name = "ImagingPlaneChannel1"
    
    expected_two_photon_series_data_shape = (30, 512, 512)
```

Maybe more readable by not needing the class method wrapper and `setUpClass`?